calibration-app/gradle.properties

```diff
@@ -28,3 +28,10 @@ android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
 
+
+# This minimal information for the maven publish plugin is required because the SDK gradle.properties is ignored when building from
+# the calibration app directory
+GROUP=org.dpppt
+POM_ARTIFACT_ID=dp3t-sdk-android
+VERSION_NAME=2.1.0
+PUBLISH_VARIANT=productionRelease
```

Comment:
Do the values of these properties matter? Or are they just required for compilation, ignoring their actual values?

Reply:
When building the calibration app, the SDK is used as a local dependency and therefore the maven-publish plugin is applied as well. However, the gradle.propreries of the SDK (containing the plugin information) is ignored and instead the one in the calibration-app is used. Therefore this is required for compilation, although it does not have any impact.